docs/recipes/Code-Splitting-Using-Holocron.md

```diff
@@ -2,6 +2,77 @@
 
 # Code Splitting Using Holocron
 
-Recipe is forthcoming.
+> To view a working sample of code splitting in One App, check out the sample module [Franks Burgers](../../prod-sample/sample-modules/franks-burgers/0.0.0/README.md)
+
+One App uses `one-app-bundler` to support dynamic importing and code splitting, out of the box.
+`one-app-bundler` uses `webpack` and `babel` (with `@babel/plugin-syntax-dynamic-import`) under
+the hood to understand the `import()` syntax and automatically split your module where ever
+the `import()` statement is used within your module. Let's go through a quick example to get
+a better grasp of code splitting.
```

Comment:
Note that holocron is code splitting and that is recommended especially for ssr. and that `import()` syntax is not supported for SSR because it creates a UI flash, so it should only be used without SSR or when triggered by a user action after the page has been rendered

Reply:
I will update to reflect what you're saying. I'll also add a note about `Suspense` and `lazy` to cover some of the spectrum of code-splitting with React.

Reply:
I've updated the docs cc27a15